ocaml/xapi-storage/generator/lib/control.ml

```diff
@@ -116,6 +122,11 @@ type volume = {
   ; keys: (string * string) list
         (** A list of key=value pairs which have been stored in the Volume
       metadata. These should not be interpreted by the Volume plugin. *)
+  ; volume_type: volume_type option [@default Some Data]
```

Comment:
While the option seems superfluous tests do not pass unless it's there

Reply:
Yes the option is a workaround to make the type check of the generated python code happy